### PR TITLE
deprecate `ui` submodule

### PR DESCRIPTION
**Pull request by ryanSoley**
closes: #165 
---

## What
  * `ui.Dashboard` now raises a `NotImplementedError` that directs users to `viz.Dashboard`
  * `viz.Dashboard` now has default widgets that replicate the old `ui.Dashboard`
  * the CLI leverages `viz.Dashboard`
  * the existing notebooks leverage `viz.Dashboard`
  * the `ui` submodule is emptied except for `ui.Dashboard` with the deprecation message, which will be removed eventually

## How to Test
  * validate that the updated notebooks run as expected
  * validate that you can launch a dashboard from the CLI
    * [these instructions](https://capitalone.github.io/rubicon-ml/dashboard.html#cli) should still be applicable

### CODE REVIEW

File: notebooks/quick-look.ipynb [hunk 1]
```diff
@@ -232,10 +232,10 @@
...
    ],
    "source": [
-    "from rubicon_ml.ui import Dashboard\n",
+    "from rubicon_ml.viz import Dashboard\n",
     "\n",
     "\n",
-    "Dashboard(rubicon=rubicon).run_server_inline()"
+    "Dashboard(project.experiments()).show()"
    ]
   },
```
Comment (shania-m) [inline notebooks/quick-look.ipynb:240]: the quicklook still has a screenshot of the old dashboard, could we update the screenshot with one of the new dashboard or just remove it all together?

File: notebooks/integrations/integration-git.ipynb [hunk 2]
```diff
@@ -117,10 +117,13 @@
     }
    ],
```
Comment (shania-m) [inline notebooks/integrations/integration-git.ipynb:118]: the notebook still has a screenshot of the old dashboard commit hash preview, could we update the screenshot with one of the new dashboard or just remove it all together?

File: notebooks/integrations/integration-prefect-workflows.ipynb [hunk 3]
```diff
@@ -647,10 +647,16 @@
     }
    ],
    "source": [
```
Comment (shania-m) [inline notebooks/integrations/integration-prefect-workflows.ipynb:649]: I get a `JSONDecodeError: Expecting value: line 1 column 1 (char 0)` on cell 7

File: notebooks/quick-look.ipynb (hunk 1 above)
Comment (ryanSoley) [reply notebooks/quick-look.ipynb:240]: good catch! I'll update the images

File: notebooks/integrations/integration-prefect-workflows.ipynb (hunk 3 above)
Comment (ryanSoley) [reply notebooks/integrations/integration-prefect-workflows.ipynb:649]: I'm gonna assume that this is a prefect issue. I can't even get prefect to run locally to test the notebook. we'll have to look into it, but its out of the scope of this

**Review by shania-m — APPROVED** [review]
LGTM 🎊 💯

**Pull request merged**